### PR TITLE
1.2/978 merge composerjson

### PR DESCRIPTION
**Pull request by charlesstrange2**
mashing centre's composer.json, packages and package config into service.

### CODE REVIEW

**Comment by katjam** [comment]
:scream: So many files!

File: bootstrap/autoload.php [hunk 1]
```diff
@@ -15,3 +15,20 @@
 */
 
 require __DIR__.'/../vendor/autoload.php';
+
+/*
+|--------------------------------------------------------------------------
+| Include The Compiled Class File
```
Comment (katjam) [inline bootstrap/autoload.php]: Cool - not seen that before.

File: composer.json [hunk 2]
```diff
@@ -17,12 +18,13 @@
         "optimus/api-consumer": "0.2.*",
         "ramsey/uuid": "^3.6",
         "sebdesign/laravel-state-machine": "^1.1",
-        "spinen/laravel-mail-assertions": "^0.2.0"
+        "spinen/laravel-mail-assertions": "^0.2.0",
+        "tom-lingham/searchy": "2.*"
```
Comment (katjam) [inline composer.json]: lol. search-y.

File: config/dompdf.php [hunk 3]
```diff
@@ -0,0 +1,266 @@
+<?php
+
+return array(
+
+    /*
+    |--------------------------------------------------------------------------
+    | Settings
```
Comment (katjam) [inline config/dompdf.php]: tl;dr assuming these are defaults?

File: config/ide-helper.php [hunk 4]
```diff
@@ -0,0 +1,179 @@
+<?php
+
+return array(
```
Comment (katjam) [inline config/ide-helper.php]: tl;dr assuming you are happy with these defaults.

File: resources/fonts/658fccc2ea95579d9a08f28d55643893.ufm [hunk 5]
```diff
@@ -0,0 +1,239 @@
+StartFontMetrics 4.1
```
Comment (katjam) [inline resources/fonts/658fccc2ea95579d9a08f28d55643893.ufm]: never heard of `.ufm` assume you need this one and the `.ufm.php` file?

Comment (charlesstrange2) [reply resources/fonts/658fccc2ea95579d9a08f28d55643893.ufm]: It's a font metric file for dompdf, for difficult fonts so you can set dompdf up with some hints.

File: config/ide-helper.php (hunk 4 above)
Comment (charlesstrange2) [reply config/ide-helper.php]: Yep

File: config/dompdf.php (hunk 3 above)
Comment (charlesstrange2) [reply config/dompdf.php]: It's the file from the ARCVStore system.

File: resources/fonts/658fccc2ea95579d9a08f28d55643893.ufm (hunk 5 above)
Comment (katjam) [reply resources/fonts/658fccc2ea95579d9a08f28d55643893.ufm]: :+1:

File: config/ide-helper.php (hunk 4 above)
Comment (katjam) [reply config/ide-helper.php]: Ace!

File: config/dompdf.php (hunk 3 above)
Comment (katjam) [reply config/dompdf.php]: :small_airplane:

**Pull request merged**